ui/client/components/graph/node-modal/editors/field/Input.js

```diff
@@ -29,7 +28,10 @@ export const Input = (props) => {
 
 Input.propTypes = {
   isMarked: PropTypes.bool,
-  value: PropTypes.string,
+  value: PropTypes.oneOfType([
```

Comment:
I wonder if it's enough? E.g. when we happen to use boolean or sth?

Reply:
There is a separate component for boolean (Checkbox)

Reply:
true